data/events.yml

```diff
@@ -0,0 +1,82 @@
+# meetup_sample:
+#   title: "Upcoming Meetup"
+#   image: false
+#   date: Month dd, yyyy
+#   event_type: Remote Event
+#   description: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque sollicitudin felis id ante placerat, at fermentum velit efficitur. Sed consequat ligula eget odio feugiat, eu condimentum diam feugiat. 
+#   label: Meetup
+#   links:
+#     twitter: https://twitter.com/SolidusIO/status/1389604484381302787?s=20 # a direct link to the post
+#     slack: https://solidusio.slack.com/archives/C03L07BUM/p1620039083115100 # a direct link to the message
+
+meetup_04062021:
```

Comment:
I'm sure you know it but writing anyway to keep track of it. Jacky's image has a John Doe in it.